packages/site-parsers/src/parsers/wix/components/document-media.js

```diff
@@ -5,15 +5,19 @@ const { createBlock } = require( '@wordpress/blocks' );
  * componentType: DocumentMedia
  */
 module.exports = {
-	parseComponent: ( component, { addMediaAttachment } ) => {
+	parseComponent: ( component, { metaData, addMediaAttachment } ) => {
 		if ( ! component.dataQuery.link ) {
 			return null;
 		}
 
-		const attachment = addMediaAttachment( {
+		const mediaUrl = metaData.serviceTopology.staticHTMLComponentUrl.replace(
+			/\/$/,
+			''
+		);
```

Comment:
I think it'd be cleaner if this were done inside the `addMediaAttachment` function as a form of normalization. That way you can always just drop in the particular string from the serviceTopology without having to think about whether it has a trailing slash or not.

Reply:
Absolutely agree.